packages/protocol/contracts/bridge/TokenVault.sol

```diff
@@ -135,17 +148,20 @@ contract TokenVault is EssentialContract {
         message.refundAddress = refundAddress;
         message.memo = memo;
 
+        require(message.callValue == 0, "V:callValue");
```

Comment:
@dantaik can you explain why this check is needed? from my understanding, the memory is initialized, and the uint256 fields `callValue` is initialized as `0`. And between the initialization and this `require` statement, there is no modification to `callValue`.

Reply:
That's true, I added this check to prevent future PRs to change the value of `callValue` without realizing it must be 0.

Reply:
ok ill add a comment for that.